languages/csharp/global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "6.0.413",
+    "version": "6.0.100",
```

Comment:
ℹ️ I may take a quick look at an SDK upgrade to 8, unless you foresee an issue.

Reply:
As long as the package remains compatible with all actively supported .NET installs. We may need to explore adding .net standard support in the future.